src/legacy/core_plugins/kibana/public/discover/kibana_services.ts

```diff
@@ -61,7 +61,6 @@ export { timezoneProvider } from 'ui/vis/lib/timezone';
 export { tabifyAggResponse } from '../../../data/public';
```

Comment:
Looks like `stateMonitorFactory` isn't used anymore.

Reply:
It isn't! thx and removed!